### PR TITLE
issue-424: remove benchmark from pythainlp

### PR DESCRIPTION
**Pull request by p16i**
Related to #428

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
 extras = {
     "attacut": ["attacut>=1.0.6"],
-    "benchmarks": ["numpy>=1.16.1", "pandas>=0.24"],
+    "benchmarks": ["numpy>=1.16.1", "pandas>=0.24", "PyYAML>=5.3.1"],
```
Comment (wannaphong) [inline setup.py:49]: Add `PyYAML>=5.3.1` to `full`

File: docs/api/benchmarks.rst [hunk 2]
```diff
@@ -19,6 +19,9 @@ Quality
 
    Qualitative evaluation of word tokenization.
 
+   **Disclaimer**: our current methodology is based on a unpublished work; hence,
```
Comment (wannaphong) [inline docs/api/benchmarks.rst:22]: Can you add to `2.1 branch`?

**Review by wannaphong — COMMENTED** [review]
Add benchmark test cli.

File: docs/api/benchmarks.rst (hunk 2 above)
Comment (p16i) [reply docs/api/benchmarks.rst:22]: sry, what do you mean?

Comment (wannaphong) [reply docs/api/benchmarks.rst:22]: > 
> 
> sry, what do you mean?

Create a pull request to add a disclaimer to the documents of 2.1.

**Pull request closed without merging**